### ISSUE TITLE
After one day of training, the perplexity and accuracy of validation set remained unchanged

### ISSUE THREAD

**Issue by lijun20** [OpenNMT/OpenNMT-py#966]
[2018-09-20 15:18:39,264 INFO] Step 380000/1000000; acc:  42.02; ppl: 55.22; xent: 4.01; lr: 0.00000; 2214/2168 tok/s;    950 sec
[2018-09-20 15:18:39,611 INFO] Loading valid dataset from data/peotry.valid.1.pt, number of examples: 7988
[2018-09-20 15:18:47,173 INFO] Validation perplexity: 56.168
[2018-09-20 15:18:47,173 INFO] Validation accuracy: 35.9623

..........

[2018-09-20 19:26:42,257 INFO] Step 509900/1000000; acc:  37.34; ppl: 55.67; xent: 4.02; lr: 0.00000; 2372/1562 tok/s;  15833 sec
[2018-09-20 19:26:46,696 INFO] Step 509950/1000000; acc:  39.09; ppl: 53.63; xent: 3.98; lr: 0.00000; 1978/2215 tok/s;  15837 sec
[2018-09-20 19:26:51,217 INFO] Step 510000/1000000; acc:  36.21; ppl: 52.56; xent: 3.96; lr: 0.00000; 2052/1240 tok/s;  15842 sec
[2018-09-20 19:26:51,299 INFO] Loading valid dataset from data/peotry.valid.1.pt, number of examples: 7988
[2018-09-20 19:26:58,682 INFO] Validation perplexity: 56.168
[2018-09-20 19:26:58,683 INFO] Validation accuracy: 35.9623

Validation perplexity: 56.168
Validation accuracy: 35.9623

train parameter：
python3 train.py -data data/nmt \
        -gpuid 3 \
	-optim adam  -learning_rate 0.001 \
	-encoder_type transformer \
        -decoder_type transformer \
        -save_model model/nmt-model
the accuracy and perplexity of validation remains unchanged. How can I solve this problem ?
